src/components/layout/Page/Page.tsx

```diff
@@ -6,14 +6,15 @@ import { Breadcrumb, CallToAction } from "components/core";
 
 import type { FlexProps, StackProps } from "@omnidev/sigil";
 import type { ActionButton, BreadcrumbRecord } from "components/core";
+import type { ReactNode } from "react";
 
 interface Props extends StackProps {
   /** Page breadcrumbs for navigation. */
   breadcrumbs?: BreadcrumbRecord[];
   /** Page header props. */
   header?: {
     /** Header section title. */
-    title: string;
+    title: string | ReactNode;
```

Comment:
`string` is a valid React node:

```suggestion
    title: ReactNode;
```

Reply:
Yeah, v obvious haha just a mind slip 😅